### PR TITLE
Fixes some incorrect packet ver dates.

### PR DESCRIPTION
**Pull request by Jedzkie**
### Pull Request Prelude

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR will be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed

Upon testing 2015-09-23fRagexe, the chat messages are also broken, but in 2015-09-16aRagexe, it does not.

Also in 2015-10-21aRagexe, this client isn't yet supports new character creation.

**Affected Branches:** master

---
This change is [Reviewable](https://reviewable.io/reviews/herculesws/hercules/1652)

### CODE REVIEW

**Comment by Jedzkie** [comment]
@MishimaHaruna @Smokexyz

**Comment by 4144** [comment]
@Jedzkie while you updating versions for chat end null pointer, can you also check for your clients input script command? I think it related to this versions. Where client add or not add null char at end of string.
Example:
```
input @var$;
mes @var$;
```

**Comment by Jedzkie** [comment]
@4144 i tried changing the client check in `clif_parse_NpcStringInput` from 20151029 to 20151021 and got this result.

input .@test;
mesf("[%d]", .@test);

result is [01], etc. depends on what number you've input, it always adds 0 before the input number.

So the current packet ver check for `clif_parse_NpcStringInput` is correct.

**Comment by 4144** [comment]
@Jedzkie it works only for strings. Ints always works.
In short old clients add always \x0 char at end.
And new clients not adding \x0 char at end.

**Comment by Jedzkie** [comment]
let me try again.

**Comment by 4144** [comment]
See check in clif_parse_NpcStringInput

**Comment by Jedzkie** [comment]
I tested on 2 clients,

2015-10-21aRagexe = no null char at end
2015-11-04aRagexe = no null char at end

I use this script

`input @var$;
mes @var$;`
